packages/@sanity/components/src/portal/Sticky.js

```diff
@@ -63,12 +70,11 @@ export default class Sticky extends React.PureComponent {
   _scrollContainerWidth = 0
 
   componentDidMount() {
-    const {
-      scrollContainer
-    } = this.props
 
-    if (scrollContainer) {
-      this.setScrollContainerElement(scrollContainer)
+    if (this.context.getScrollContainer) {
+      this.setScrollContainerElement(this.context.getScrollContainer())
+    } else {
+      this.setScrollContainerElement(document.body)
```

Comment:
Can `context.scrollContainer()` return null/undefined?
This could be made more robust and concise with:
```js
const {getScrollContainer} = this.context
const scrollContainer = (typeof getScrollContainer === 'function' && getScrollContainer()) || document.body
this.setScrollContainerElement(scrollContainer)
```

Reply:
👍